fpga_interchange/pseudo_pip_model.cc

```diff
@@ -440,7 +448,7 @@ void PseudoPipModel::update_site(const Context *ctx, size_t site)
         // that pseudo pip violates the LUT element equation.
         for (const PseudoPipBel &bel : bels) {
             const BelInfoPOD &bel_data = type_data.bel_data[bel.bel_index];
-            if (bel_data.lut_element == -1) {
+            if (bel_data.lut_element == -1 || bel.input_bel_pin == -1) {
```

Comment:
I am not sure about this, but shouldn't the no-input pseudo cell be checked as well?

Reply:
hmm, possibly, I think I need to understand how `lut_wires_unavailable` works better

Reply:
I've now added a check that in this case, any of the bel pins is not blocked by the LUT equation, which I think is at least more correct than no check at all.